addOns/websocket/src/main/zapHomeFiles/scripts/scripts/websocketpassive/application_error_scanner_regex.js

```diff
@@ -0,0 +1,73 @@
+// * This Script analyze incoming websocket messages for error messages
+
+// * Based on org.zaproxy.zap.extension.pscanrules.ApplicationErrorScanner
+// * Application error strings are equal to (characters '/' is escaped -> '//'):
+// ** https://github.com/zaproxy/zap-extensions/blob/master/addOns/pscanrules/src/main/resources/org/zaproxy/zap/extension/pscanrules/resources/application_errors.xml
```

Comment:
Perhaps this one should have a comment that differentiates it from the one above?

Reply:
Fixed in cd5c145